slang-antlr/src/test/java/com/sonarsource/slang/antlr/SLangConverterTest.java

```diff
@@ -472,6 +474,79 @@ public void stringLiterals() {
     }
   }
 
+  @Test
+  public void jump() {
+    String slangCode = "break foo";
+    Tree tree = converter.parse(slangCode + ";");
```

Comment:
IMO all this hustle around `TopLevelTree` is not required, better smth like this `JumpTree jumpTree = (JumpTree) converter.parse("...").children().get(0);`